syntax = "proto3";

message ISearchRequest {
    string login = 1;
}
